public/configurations/visualizations/aar-enterprise-app-pie.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "aar-enterprise-app-pie",
```

Comment:
Name of this file should be "aar-enterprise-all-app-usage.json" or something which is more meaningful